### PR TITLE
added products db model

### PR DESCRIPTION
**Pull request by p-mcgrath**

### CODE REVIEW

File: server/db/models/Product.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    allowNull: false,
+    validate: {
+      notEmpty: true,
+    },
+  },
+  description: {
+    type: Sequelize.TEXT,
+  },
+  price: {
+    type: Sequelize.FLOAT,
```
Comment (ariadosic) [inline server/db/models/Product.js:17]: Sequelize.Decimal(10,2) ???

File: server/db/models/Product.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  discount: {
+    type: Sequelize.FLOAT,
+    defaultValue: 0,
+    validate: {
+      min: 0,
+      max: 1,
+    },
+  },
+  displayPrice: {
+    type: Sequelize.FLOAT,
```
Comment (ariadosic) [inline server/db/models/Product.js:32]: Sequelize.Decimal(10,2) ???

**Review by ariadosic — CHANGES_REQUESTED** [review]
good job :)

File: server/db/models/Product.js (hunk 2 above)
Comment (p-mcgrath) [reply server/db/models/Product.js]: fixed

**Review by ariadosic — APPROVED** [review]

**Pull request merged**